cmd/cluster/openstack/create.go

```diff
@@ -225,3 +223,33 @@ func NewCreateCommand(opts *core.RawCreateOptions) *cobra.Command {
 
 	return cmd
 }
+
+// validateOpenStackCredentialsFile checks that the OpenStack credentials file exists
+// and that the cloud name is "openstack" which we hardcode for now.
+func validateOpenStackCredentialsFile(credentialsFile string) error {
+	if credentialsFile == "" {
+		return fmt.Errorf("OpenStack credentials file is required")
+	}
+
+	if _, err := os.Stat(credentialsFile); err != nil {
+		return fmt.Errorf("OpenStack credentials file does not exist: %w", err)
+	}
+
+	cloudsFile, err := os.ReadFile(credentialsFile)
+	if err != nil {
+		return fmt.Errorf("failed to read OpenStack credentials file: %w", err)
+	}
+	clouds := make(map[string]interface{})
+	if err := yaml.Unmarshal(cloudsFile, &clouds); err != nil {
+		return fmt.Errorf("failed to parse OpenStack credentials file: %w", err)
+	}
+	_, ok := clouds["clouds"]
+	if !ok {
+		return fmt.Errorf("'clouds' key not found in credentials file")
+	}
+	clouds = clouds["clouds"].(map[string]interface{})
+	if _, ok := clouds["openstack"]; !ok {
```

Comment:
why do we have the openstack cloud name as a requirement?

Reply:
because we need to know what is the cloud name to take from clouds.yaml, so for now we hardcode it to "openstack". One option is to add "cloudName" to the HCP.Spec.OpenStack (and CLI) to allow users to change it.